### PR TITLE
feat(synthetics): Removed old IP and inserted new IP ranges, new call out, new metadescription

### PR DESCRIPTION
**Pull request by akristen**
To do:

~- Make table more readable~ 
~- Insert `<br>` (or equivalent) to make breaks after listed IPs~ 
~- x2 check that the IPs match info given to me~
~- Write better metaDescription~
~- Ask SME if I should link to DNS information in lieu of listing DNS addresses~
~- Request review from SME~
- Ask tw for feedback on table readability

### CODE REVIEW

File: src/content/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips.mdx [hunk 1]
```diff
@@ -1,289 +1,291 @@
...
   - Synthetic monitoring
   - Administration
 translate:
   - jp
-metaDescription: 'For New Relic synthetic minions: upcoming IP addresses to be implemented in July 2021.'
+metaDescription: 'New runtime capabilities for synthetic monitoring may require you update your firewall allow list to accept network requests from new IP address ranges'
 ---
 
-On July 15 2021, we'll be adding new IP addresses for several synthetics locations for both [US locations](#us_new_ip) and [EU locations](#eu_new_ip). To ensure your monitors are not affected by these changes, please add the appropriate IP addresses to your firewalls allow list.
+By the end of March 2022, an updated runtime will be launched to support newer versions of Chrome. As a result, you'll need to reconfigure your firewall's allow list, bot detection software, and other related applications to accept network requests from new IP address ranges.
```
Comment (bradleycamacho) [inline src/content/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips.mdx:12]: I want to point out that we tend to avoid "dating" changes and docs. It's a pain to keep dated info current and accurate.

In this case, I _think_ it's fine depending on the strategy here. Are customers aware this is the doc they should look at for this info? Will they be notified in some other form? I worry that just updating an already existing doc won't really serve as a good enough form of communication about these changes.

File: src/content/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips.mdx [hunk 2]
```diff
@@ -1,289 +1,291 @@
...
 ---
 
-On July 15 2021, we'll be adding new IP addresses for several synthetics locations for both [US locations](#us_new_ip) and [EU locations](#eu_new_ip). To ensure your monitors are not affected by these changes, please add the appropriate IP addresses to your firewalls allow list.
+By the end of March 2022, an updated runtime will be launched to support newer versions of Chrome. As a result, you'll need to reconfigure your firewall's allow list, bot detection software, and other related applications to accept network requests from new IP address ranges. 
 
-For the current list of IP addresses and more about this topic, see [Synthetic monitor public minion IPs](/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips/).
+Monitors created after the new runtime launch will use the new IP address space. Existing monitors will be moved to the new IP address space later this year. Check out [Synthetic monitor public minion IPs](/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips/) for more information about IP addresses and synthetic monitoring.
```
Comment (bradleycamacho) [inline src/content/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips.mdx:14]:
```suggestion
Monitors created after the new runtime launch will use the new IP address space. Existing monitors will be moved to the new IP address space in the future. Check out [Synthetic monitor public minion IPs](/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips/) for more information about IP addresses and synthetic monitoring.
```

File: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx [hunk 3]
```diff
@@ -14,9 +14,11 @@ redirects:
   - /docs/synthetics/synthetic-public-minion-ips
 ---
 
-New Relic uses a group of minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed in different data centers around the globe, and they are in charge of actually running your monitors.
+You can set up minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed from different data centers around the globe and are in charge of running your monitors.
```
Comment (bradleycamacho) [inline src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:17]:
```suggestion
You can set up minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed from different data centers around the globe and run your monitors.
```

File: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx [hunk 4]
```diff
@@ -14,9 +14,11 @@ redirects:
...
 
-Because of this, ensure your firewall allows their network requests through it. Minion IP addresses are publicly available in JSON format, so you can easily read and parse them.
+Because of this, ensure your firewall allows your minions to make network requests through it. Minion IP addresses are publicly available in JSON format, so you can easily read and parse them.
```
Comment (bradleycamacho) [inline src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:19]: Where can users find these publicly available addresses? If it's in this doc, link over to it.

File: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx [hunk 5]
```diff
@@ -25,19 +27,15 @@ Because of this, ensure your firewall allows their network requests through it.
 
 ## IP addresses are not personal data [#not-personal-data]
 
-Minions are deployed on servers, and the agents are expected to be activated using non-personal credentials. IP addresses associated with minions running on servers are not personal data under data protection and privacy laws. For more information, see the [Synthetic's security documentation](/docs/synthetics/new-relic-synthetics/getting-started/security-new-relic-synthetics).
+Minions are deployed on servers, and the agents are activated using non-personal credentials. IP addresses associated with minions running on servers are not personal data under data protection and privacy laws. For more information, see the [Synthetic's security documentation](/docs/synthetics/new-relic-synthetics/getting-started/security-new-relic-synthetics).
```
Comment (bradleycamacho) [inline src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:30]: This wording can change the meaning pretty significantly. Do the users provide the credentials? If so, I would probably keep "expected to be" as it implies the best practice is to use non-personal credentials. If we control the credentials, this edit is fine.

File: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx [hunk 6]
```diff
@@ -25,19 +27,15 @@ Because of this, ensure your firewall allows their network requests through it.
...
 IP addresses for released locations are subject to change. If a change is needed, we'll attempt to proactively notify customers prior to any changes via e-mail. You can also check the [Explorers Hub](https://discuss.newrelic.com/) for updates.
 
 <Callout variant="important">
-   Synthetics is [adding new IP addresses on July 15 2021](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips).
-</Callout>
-
-<Callout variant="tip">
-  In the S3 URL paths for this feature, `production` represents US-based accounts and `eu` represents [EU-based accounts](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers).
-</Callout>
+    By the end of March 2022, [new IP address ranges](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips) may require that you reconfigure your firewall settings.
```
Comment (bradleycamacho) [inline src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:37]: I'd get as granular as you can here. This is fine for now, but I'd update it to the exact date as soon as it's known.

Comment (bradleycamacho) [inline src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:37]: I'd also be a bit more specific about what would entail "may" becoming "will". I'd specify in which instances someone would have to reconfigure.

File: src/content/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips.mdx (hunk 1 above)
Comment (bradleycamacho) [inline src/content/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips.mdx:12]: See comment way down below, but I'd update the date as soon as you know the exact launch day.

**Review by bradleycamacho — CHANGES_REQUESTED** [review]
Looks really good other than a few comments!

File: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx (hunk 6 above)
Comment (akristen) [reply src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:37]: "May" was to account for the short period that both runtimes would be supported for already-existing monitors. Users won't have to update their allow lists until:

- They create a new monitor after the new runtime launch
- The old runtime (and their associated / old IPs) is no longer supported. (No date for this yet) 

At that point, they'll have to update their allow lists. The call out is mostly just to notify users that this is in the works to keep an eye out for.

File: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx (hunk 5 above)
Comment (akristen) [reply src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:30]: !! Good catch! I'll change it back to 'expected to be.' I removed it to streamline the sentence and didn't realize meaning would be affected ;)

File: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx (hunk 3 above)
Comment (akristen) [reply src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:17]: How do we feel about: 

> You can set up minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions run your monitors and are deployed from different data centers around the globe.

File: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx (hunk 6 above)
Comment (bpeckNR) [reply src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:37]: We don't want to remove this callout about production vs eu in the JSON listings. While the new IP address ranges will include both our US and EU locations, the existing IPs will continue to be used for months too and this production / eu breakdown is important for those IPs.

File: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx (hunk 3 above)
Comment (bpeckNR) [reply src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:17]: The original wording is less confusing to me. When we mention that customers can set up minions it will cause confusion between public and private minions. We create and maintain minions for our public locations where customers are responsible for deploying minions only for private locations behind the firewall.

```suggestion
New Relic maintains public minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed in different data centers around the globe and are in charge of running your monitors.
```

File: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx (hunk 4 above)
Comment (bpeckNR) [reply src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:19]: We will cause confusion between public and private minions if we refer to these as your minions. 
```suggestion
Because of this, ensure your firewall allows synthetics network requests. Minion IP addresses are publicly available in JSON format, so you can easily read and parse them.
```

**Review by bpeckNR — CHANGES_REQUESTED** [review]
I suggested a few minor changes.

**Comment by bradleycamacho** [comment]
@akristen another though: is this meant to serve as a temporary transition guide, or will this be a source of truth for the forseeable future? If it is, I would take another pass at the working and tenses. Instead of framing it as "We will update the ips, which means you will need to do xyz", maybe we should reframe it as "This doc covers transitioned from pre-2022 ips to current ips.".

File: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx [hunk 7]
```diff
@@ -14,30 +14,27 @@ redirects:
   - /docs/synthetics/synthetic-public-minion-ips
 ---
 
-New Relic uses a group of minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed in different data centers around the globe, and they are in charge of actually running your monitors.
+New Relic uses public minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed in different data centers around the globe and are in charge of running your monitors.
 
-Because of this, ensure your firewall allows their network requests through it. Minion IP addresses are publicly available in JSON format, so you can easily read and parse them.
+Because of this, ensure your firewall allows synthetics network requests. Minion IP addresses are publicly available in JSON format, so you can easily read and parse them.
```
Comment (bradleycamacho) [inline src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:19]: Should this be `synthetic network request`?

Comment (bradleycamacho) [reply src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:19]: Genuinely not sure what the terminology we use here is.

File: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx [hunk 8]
```diff
@@ -14,30 +14,27 @@ redirects:
...
 
-**Recommendation:** For easier maintenance, use these options:
+Keep in mind that:
```
Comment (bradleycamacho) [inline src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx:21]: Try to be more prescriptive here. A good rule of thumb is to avoid "You can" or "You should" when recommending options. Maybe rephrase this optional steps or something similar.

**Review by bradleycamacho — DISMISSED** [review]
Looks good other than a few small comments! Great job Alexa!

File: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx (hunk 8 above)
Comment (akristen) [reply src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx]: Went with optional steps. When I redo this doc, I'm going to revisit this tbh.
